tests/5.0/target_parallel_loop/test_target_parallel_loop_reduction.c

```diff
@@ -0,0 +1,112 @@
+//===-------------------test_target_parallel_loop_reduction.c-----------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// Description - This is a test program to demonstrate how reduction
+// min/max/+ clauses are used with target parallel loop construct. 
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+#define N 1024
+
+int testMin() {
+  OMPVV_INFOMSG("testMin");
+  int a[N];
+  int b[N];
+  int errors = 0;
+  srand(1);
+
+  for (int i = 0; i < N; ++i) {
+    a[i] = (int) rand() / (double)(RAND_MAX / 100);
+    b[i] = (int) rand() / (double)(RAND_MAX / 100);
+  }
+
+  int result = a[0] + b[0];
+
+#pragma omp target parallel loop reduction(min:result) map(to: a[0:N], b[0:N]) map(tofrom: result)
+    for (int i = 0; i < N; ++i) {
+      result = fmin(a[i] + b[i], result);
+    }
+
+  int hostMin = a[0] + b[0];
+  for (int i = 0; i < N; ++i) {
+    hostMin = fmin(hostMin, a[i] + b[i]);
+  }
+
+  OMPVV_TEST_AND_SET_VERBOSE(errors, result != hostMin);
+  OMPVV_ERROR_IF(host_min != result, "Minimum value on device is %d but expected minimum from host is %d.", result, hostMin);
```

Comment:
Change `host_min` to `hostMin`

Reply:
I think, this 'OMPVV_ERROR_IF' is not required as already used in the definition of 'OMPVV_TEST_AND_SET_VERBOSE' .

Reply:
`OMPVV_ERROR_IF` may be still useful in that it can provide detailed error messages (`OMPVV_TEST_AND_SET_VERBOSE` just tells the existence of the error, without any detailed information.)